### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

File: Gemfile.lock [hunk 1]
```diff
@@ -85,6 +85,7 @@ GEM
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.1)
+    mysql2 (0.5.3)
```
Comment (ngocvt-0484) [inline Gemfile.lock:88]: Why hasn't gem file changed but gemfile.lock changed ?

**Comment by ngocvt-0484** [comment]
please delete fix file

File: Gemfile.lock (hunk 1 above)
Comment (stephanie1997-kek) [reply Gemfile.lock:88]: image
image
i got some conflict after i pull request to github. so i type this coding to fix the conflict. after that my static page home and help is disappear. I think is this coding make the gemfile.lock changed

**Comment by stephanie1997-kek** [comment]
> please delete fix file

Can you tell me how to delete fix file?

**Comment by stephanie1997-kek** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**